src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp

```diff
@@ -16,4 +18,15 @@ namespace intrp {
 double predicted_zero_crossing_value(const std::vector<double>& x_values,
                                      const std::vector<double>& y_values);
 
+/*!
+ * \brief Predicts the zero crossing of multiple functions contained in a vector
+ * of datavectors.
+ *
+ * Fits a set of N linear functions where N is the size of y_values that use the
```

Comment:
comma after "functions"

Isn't N the size of the DataVector, not the size of y_values? The size of x_values and y_values are the size of the std::vectors<>, which are the number of points used in each fit. N is the number of fits, which is the DataVector size, right?